containers/src/Data/Set/Internal.hs

```diff
@@ -897,6 +905,24 @@ intersection t1@(Bin _ x l1 r1) t2
 {-# INLINABLE intersection #-}
 #endif
 
+#if (MIN_VERSION_base(4,9,0))
+-- | The intersection of a series of sets. Intersections are performed left-to-right.
```

Comment:
Please mention the limitation to `base >= 4.9` in the docs.

Reply:
Do we need `intersections`? Isn't it just `sconcat`? I'm not 100% allergic to conditional exports (we have pattern synonyms for sequences), but I'd generally prefer to avoid them.